tests/js/server/aql/aql-queries-geo.js

```diff
@@ -100,10 +100,6 @@ function ahuacatlLegacyGeoTestSuite () {
       db._drop("UnitTestsAhuacatlLocationsNon");
     },
 
-    testLegacyNearInvalidCoordinate : function () {
```

Comment:
Why was this test removed?

Reply:
Because 

> Fix S2LatLng -> S2Point works only for valid S2LatLng:
It was really strange. In some cases we produce error to user. In other ignore and works incorrect. In third makes normalize and works correct.
I think always produce error is better, but in such case we need way to update old user data. It's impossible now, so I think better to always normalize
So I change behavior: In some queries user couldn't specify not normalized points. It was error/invalid behavior, now they can and it will be valid

Reply:
I found it. There is a subtle change in the geo index, which now normalizes points (in the non-geojson case) rather than rejecting them. I think this is a good change, I approve of it. I now understand why the tests had to be changed.